mmv1/test/validate_third_party_test.go

```diff
@@ -14,10 +14,10 @@ func TestTemplatesStillNeedToBeTemplates(t *testing.T) {
 	if !ok {
 		t.Fatal("Failed to get current test file path")
 	}
-	testDir := filepath.Dir(testFilePath)
+	thirdPartyDir := filepath.Join(filepath.Dir(filepath.Dir(testFilePath)), "third_party")
```

Comment:
It looks like this is adding another `filepath.Dir` call compared to the previous implemementation; why isn't that causing the implementation to fail?